app/config/routing.yml

```diff
@@ -96,3 +96,12 @@ app_redirecting:
     host:     "%app_host%"
     resource: "@AppBundle/Controller/RedirectingController.php"
     type:     "annotation"
+
+# This route must be loaded the last to prevent an override
+app_static_page:
+    host:    "%app_host%"
+    path:    /s/{slug}
```

Comment:
We really want this weird path ? Looks bad for SEO no ?

Reply:
There is already a route with `/{url}` for path. To prevent some errors I prefer use this url and it is write in the ticket of this feature ;)